.github/workflows/kind.yml

```diff
@@ -71,7 +71,22 @@ jobs:
         sudo mv kind /usr/local/bin
     - name: Run e2e tests
       run: |
-        ./ci/kind/test-e2e-kind.sh --encap-mode encap
+        ./ci/kind/test-e2e-kind.sh --encap-mode encap | tee kind-test.txt
+        echo "::set-output name=log_dir::$(cat kind-test.txt | grep -o "Test logs (if any) will be exported under the '[^\"]*'" | awk '{print $10}' | tr -d "'")"
```

Comment:
this is a bit of an overkill: can we leverage the `--logs-export-dir` option instead which is supported by e2e tests and lets you provide a specific directory for logs?

Reply:
then you can use `if: ${{ failure() }}` in the next step to determine whether or not to upload the logs